setup.py

```diff
@@ -66,7 +66,56 @@
         "wheel==0.36.2",
         "twine==1.14.0",
         "check-wheel-contents==0.2.0",
-    ]
+    ],
+
+    # All dependencies above.
+    # NOTE: No need to manually remove the duplicates, setuptools automatically
+    # does that.
+    "all": [
```

Comment:
I think it should just be the `visualize` deps. Then we may add extra deps in the future.